### DISCUSSION TITLE
Stuttering occurs when the menu is opened/closed.

### DISCUSSION THREAD

**Discussion by TamerCgn** [IBAX-io/go-ibax#2105] [Bug Submission]
Stuttering occurs when the menu is opened/closed.
In addition, when the menu is closed, the texts are intertwined.

https://user-images.githubusercontent.com/95222290/219406493-abd6f146-c871-4732-98e0-dcddb8c12c1b.mov

macOS ventura 13.0

**Comment by Bleiddos89** [comment]
!

**Reply by Suneel081** [reply]
Good

**Reply by Lockelamora8** [reply]
Nice

**Reply by Dosarii** [reply]
Follow with interest

**Reply by cipetpet1** [reply]
We must be different, more dynamic

**Reply by reedkaster** [reply]
nice

**Reply by Aditya1o** [reply]
Nice project

**Comment by trgttrgt** [comment]
..

**Reply by ysfcngz1** [reply]
uygulamada çalışır drumda

**Comment by ysfcngz1** [comment]
ara yüz geliştirme devam etmek zorunda

**Comment by Officialindige** [comment]
Nice interface

**Reply by reedkaster** [reply]
yes

**Comment by zeland2323** [comment]
Thanks

**Reply by reedkaster** [reply]
u'r welcome

**Comment by cipetpet1** [comment]
We must be different, more dynamic

**Comment by orjinim** [comment]
selam

**Comment by volky1983** [comment]
> Menü açılıp/kapandığında kekemelik meydana geliyor. Ayrıca menü kapatıldığında metinler iç içe geçiyor.
> 
>  Ekran.Kaydi.2023-02-16.18.04.42.mov 
> macOS ventura 13.0

ok
